test/NameBinding/dynamic-callable-silgen.swift

```diff
@@ -0,0 +1,25 @@
+// RUN: %target-swift-frontend -emit-silgen -verify %s | %FileCheck %s
```

Comment:
SILGen tests should be located in test/SILGen/

Reply:
Done in https://github.com/apple/swift/pull/16980/commits/1329ba1ad003d1706a66dd298d2079ea24f57d81.